Tests.NetCore/TextSerializerTests.cs

```diff
@@ -0,0 +1,197 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Prometheus.Tests;
+
+[TestClass]
+public class TextSerializerTests
```

Comment:
The typical pattern we use is that every test class is also present in the Tests.NetFramework project via "include existing file", to ensure that the same logic also works under .NET Framework (if it is not .NET Core specific).